audbackend/core/backend.py

```diff
@@ -398,23 +356,21 @@ def put_archive(
         for file in files:
             path = os.path.join(src_root, file)
             if not os.path.exists(path):
-                raise FileNotFoundError(
-                    errno.ENOENT, os.strerror(errno.ENOENT), path,
-                )
+                utils.raise_file_not_found_error(path)
 
         with tempfile.TemporaryDirectory(dir=tmp_root) as tmp:
-            _, archive_name = self.split(dst_path)
-            archive = audeer.path(tmp, f'{archive_name}-{version}.zip')
+
+            archive = audeer.path(tmp, os.path.basename(dst_path))
```

Comment:
We need to update `setup.cfg` to require version `1.18.0` as we added `audeer.path()` there.

Reply:
done